app/models/orphan.rb

```diff
@@ -134,11 +136,15 @@ def sponsored_siblings_does_not_exceed_siblings_count
   end
 
   def default_sponsorship_status_unsponsored
-    self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    if new_record?
+      self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    end
   end
 
   def default_orphan_status_active
-     self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    if new_record?
+      self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    end
```

Comment:
Agreed, but we already have a method that does that, `after_initialize`.
To do as you are proposing we could override `after_initialize` as a method, and merge the code from the three methods into one single method, but I think that loses clarity compatred to the method call at the top of the class we currently have.
But to create another method that has the same responsibility that `after_initialize` has, that `after_initialize` calls, I don't see value in.